### ISSUE TITLE
Build a machine learning engine with better accuracy

### ISSUE THREAD

**Issue by amjcosta** [codeforgoodconf/black_holes_backend#2]
It would be ideal to create an engine with greater than 95% accuracy. One way to accomplish this may be using tensorflow to build a neural net.

**Comment by amjcosta** [comment]
@rlbyrne I heard you would like to dig into neural nets as well? Feel free to add yourself :smiley:

**Comment by frankamp** [comment]
The current error stats are: 
```
Correlation coefficient                  0.5191
Mean absolute error                      0.1548
Root mean squared error                  0.2904
Relative absolute error                 96.3712 %
Root relative squared error            102.4633 %
Total Number of Instances              625     
```
If a new ML engine/choice could be plugged in, these are the current numbers to beat. We did range as high as 70% correlation, but I also think that the lack of negative training examples are the main factor hurting the current effort.

**Comment by sauln** [comment]
Kamino closed and cloned this issue to codeforgoodconf/black_holes
